### DISCUSSION TITLE
How to use Google Sans after v0.9.8?

### DISCUSSION THREAD

**Discussion by jcperil** [ReadYouApp/ReadYou#402] [Q&A] [answered]
I understand the decision to remove Google Sans. I don't think there is any risk if few individuals choose to use Google Sans in Read You, right?

Can you please answer how one can use this font after v0.9.8 update? Do I need to download this font from some place?

**Comment by Ashinch** [comment, marked as answer]
If you have obtained the ttf file of GoogleSans from somewhere (such as a search engine), you can apply it anywhere.
But I'm sorry, I will not provide you with any related information here.
